### DISCUSSION TITLE
I need help on figuring out how to integrate your services with my wordpress site. I also need help on figuring out how to install your code into my c++ script for the product I'm selling.

### DISCUSSION THREAD

**Discussion by AltCoinWiz** [Labs64/NetLicensing-Community#107] [Questions] [answered]
**Question**
Ask question about NetLicensing services and functionality here.

**Additional context**
Add any other context or screenshots related to this question.

**Comment by r-brown** [comment, marked as answer]
Hello @Eddiem440,

We provide [client libraries for the NetLicensing integration](https://netlicensing.io/wiki/client-libraries) for the various tech stacks and programming languages. These include PHP (WordPress) and C++

There are some public community integrations available either, such as [philipplive/netlicensing](https://github.com/philipplive/netlicensing)

Feel free to use these projects as it is or the foundation for your individual implementation.
